cmd/flags/admin.go

```diff
@@ -138,5 +169,6 @@ func (s *AdminServer) registerPprofHandlers() {
 
 // Close stops the HTTP server
 func (s *AdminServer) Close() error {
+	_ = s.tlsCertWatcherCloser.Close()
```

Comment:
nit: you can use multierr here to combine the two errors and return as one

Reply:
In collectors, we can read that
> watchers actually never return errors from Close

so, I'm not sure what multierr will bring in here ?